src/map/Map.js

```diff
@@ -611,6 +612,8 @@ L.Map = L.Evented.extend({
 		if (this.options.trackResize) {
 			L.DomEvent[onOff](window, 'resize', this._onResize, this);
 		}
+
+		this[onOff]('moveend', this._onMoveEnd);

```

Comment:
So like this https://github.com/Leaflet/Leaflet/pull/3831/files#diff-607b516b5121104eef3007cb5727e69cR616 ? :)